<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>885</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>P2P Messaging</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="logTextBox"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLineEdit" name="peerIDInput">
             <property name="placeholderText">
              <string>Peer ID</string>
             </property>
            </widget>
           </item>
      <item>
       <widget class="QLineEdit" name="peerIPInput">
        <property name="placeholderText">
         <string>Peer IP</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="peerPortInput">
        <property name="placeholderText">
         <string>Peer Port</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="messageInput">
        <property name="placeholderText">
         <string>Type your message here...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sendButton">
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
